pkgs/top-level/all-packages.nix

```diff
@@ -27989,6 +27989,16 @@ with pkgs;
 
   prusa-slicer = callPackage ../applications/misc/prusa-slicer { };
 
+  snapmaker-luban = callPackage ../applications/misc/snapmaker-luban {
+    branch = "stable";
```

Comment:
Why do you pass in branch and then within access via `.${branch}`, can you not just return the attribute set and access here? See `super-slicer-staging = (callPackage ../applications/misc/prusa-slicer/super-slicer.nix { }).staging;` just below

Reply:
Ah, I was using discords derivation as an example. Anyways, since we should not package betas this file is removed.